From 0e5f9ba58b22957bea44ca2b95e354ad2f6e66ed Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 25 Jan 2023 21:25:09 +0100
Subject: [PATCH 35/41] Revert "Revert "mm/compaction: fix set skip in
 fast_find_migrateblock""

This reverts commit 3d5d57bca1ae9c1445cf967d10214009deab4825.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 mm/compaction.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mm/compaction.c b/mm/compaction.c
index b4398527b..f7ffadeb3 100644
--- a/mm/compaction.c
+++ b/mm/compaction.c
@@ -1845,7 +1845,6 @@ static unsigned long fast_find_migrateblock(struct compact_control *cc)
 					pfn = cc->zone->zone_start_pfn;
 				cc->fast_search_fail = 0;
 				found_block = true;
-				set_pageblock_skip(freepage);
 				break;
 			}
 		}
-- 
2.39.1.388.g2fc9e9ca3c

